test/instrumentation.ts

```diff
@@ -51,6 +51,27 @@ describe('instrumentation_info', () => {
     );
   });
 
+  it('should set library version to unknown', () => {
+    const data = {some: 'value'};
+    const entry = instrumentation.createDiagnosticEntry(
+      undefined,
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      data as any
+    ) as Entry;
+    assert.equal(
+      entry.data?.[instrumentation.DIAGNOSTIC_INFO_KEY]?.[
+        instrumentation.INSTRUMENTATION_SOURCE_KEY
+      ]?.[0]?.[NAME],
+      instrumentation.NODEJS_LIBRARY_NAME_PREFIX
+    );
+    assert.equal(
+      entry.data?.[instrumentation.DIAGNOSTIC_INFO_KEY]?.[
+        instrumentation.INSTRUMENTATION_SOURCE_KEY
+      ]?.[0]?.[VERSION],
+      'unknown'
+    );
+  });
+
```

Comment:
It would be nice to also have a test for retrieving the `version` value from a `package.json` file here.

Reply:
Yep, there is one - see [here](https://github.com/googleapis/nodejs-logging/blob/42859ab634721cb57ca76960f8a8aa5f6dd3a987/test/instrumentation.ts#L35)

Reply:
oh that's great!